lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -162,7 +162,7 @@
 
 .slideshow__controller {
   display: table;
-  margin: 0 auto;
+  margin: $base-spacing auto;
```

Comment:
Using padding for the vertical spacing breaks the controller
![screen shot 2016-09-21 at 11 16 23 am](https://cloud.githubusercontent.com/assets/897368/18717108/e820bbd2-7fec-11e6-8839-d1395e1297a7.png)

The buttons are positioned absolutely so the padding has no effect